readme.md

```diff
@@ -495,6 +495,16 @@
 | ------: | -------: | -----: |
 |   1 - 5 |       10 | 45-60m |
 
+###  [Terraforming Mars](https://en.wikipedia.org/wiki/Terraforming_Mars_(board_game))
+
+> Terraforming Mars is a game where players take the role of corporations to terraform the planet by raising the temperature and creating oxygen, water, plant and animal life. Players compete to earn the most points, which are measured by their contribution to terraforming, as well as building human infrastructure.
+
+![Terraforming Mars](https://upload.wikimedia.org/wikipedia/en/f/f0/Terraforming_Mars_board_game_box_cover.jpg)
+
+| Players | Min. Age | Time |
+| ------: | -------: | -----: |
+|  1 - 5  |       12 |   120m |
```

Comment:
The linter, in the Travis CI checks, is complaining about the markdown formatting for the table here. Corrected, it would look like:

```
| Players | Min. Age | Time |
| ------: | -------: | ---: |
|   1 - 5 |       12 | 120m |
```